### PR TITLE
Fix the lang flag

### PR DESCRIPTION
**Pull request by cdemetriadis**

### CODE REVIEW

File: lib/genMixin.js [hunk 1]
```diff
@@ -6,14 +6,17 @@ const readFile = util.promisify(fs.readFile)
 const writeFile = util.promisify(fs.writeFile)
 
 async function generateMixin(styles, lang, outputFolder) {
+    let templateFile
+    let ext
+    let name
```
Comment (kerrmarin) [inline lib/genMixin.js:11]: I wonder if these should have initial values? i.e. `= ""`

Comment (cdemetriadis) [reply lib/genMixin.js:11]: They could be the default settings. The ones that appear in the else () statement.

Comment (kerrmarin) [reply lib/genMixin.js:11]: Yeah that would probably work? Alternatively, make a function:

```js
const getStylesInformation = (lang) => {
     if (lang == "less") {
        return {
            templateFile: "/../lib/template.less.hb"
            ext: "less",
            name: "LESS"
    }
    } else { // if lang == "sass"
        return {
            templateFile: "/../lib/template.scss.hb",
            ext: "scss",
            name: "SASS"
        }
    }
}
```

and return that then use the return from that function?

**Comment by cdemetriadis** [comment]
@kerrmarin I moved It to the shared lib. Looks cleaner now. Thanks!

File: lib/genMixin.js [hunk 2]
```diff
@@ -6,18 +7,11 @@ const readFile = util.promisify(fs.readFile)
...
-        name = "LESS"
-    } else { // if lang == "sass"
-        templateFile = `/../lib/template.scss.hb`
-        ext = "scss"
-        name = "SASS"
-    }
+    const styleInformation = getStyleInformation(lang)
+    let templateFile = styleInformation.templateFile
+    let ext = styleInformation.ext
+    let name = styleInformation.name
```
Comment (kerrmarin) [inline lib/genMixin.js:13]: These could now be `const`, or if you don't want them here you could use `styleInformation.XX` directly where you are using these.

**Review by kerrmarin — APPROVED** [review]

File: lib/genMixin.js (hunk 2 above)
Comment (cdemetriadis) [reply lib/genMixin.js]: Oh, sure. I removed them...

**Review by atomoil — APPROVED** [review]

**Pull request merged**